Readme.md

```diff
@@ -0,0 +1,799 @@
+# Minsky
+
+Dynamic flow systems program to simulate economic flows with complex system dynamics.
+
+## Getting Started
+
+## System requirements
+Minsky is an open source program available for: 
+- Windows
+- Mac OS X,
+- various Linux distributions
+
+Also compilable on any suitable Posix compliant system. 
+Go to our [SourceForge](http://minsky.sf.net) to download a version 
+
+Linux packages are available from the [OpenSUSE build service]
+(https://build.opensuse.org/package/show/home:hpcoder1/minsky)
+
+## Getting help
+Press the F1 key, or select ``help'' from the context menu. Help is
+context-sensitive.
+
+### Components of the Program
```

Comment:
I'm inclined to think that anything after this point is a little too detailed for a README file. 

Since this is a top level web page, I would have thought something like a copy of what appears on https://minsky.sf.net might be more appropriate. Since we're going to be using both SourceForge and Github, the top level readmes need to be fairly short so they can be kept in sync. One difference between the sites is that the SourceForge site caters for end users (file release system, tickets, the manual - which currently is not available, it appears, mailing lists), whereas Github caters for developers (code repo, developer documentation). Much of the SF wiki could be ported over to Github, as it is mostly developer documentation.